vumi/persist/riak_manager.py

```diff
@@ -293,16 +293,10 @@ def _search_iteration(self, bucket, query, rows, start):
 
     def real_search(self, modelcls, query, rows=None, start=None):
         rows = 1000 if rows is None else rows
+        start = 0 if start is None else start
```

Comment:
If this were a new method, I'd completely agree. What we're doing here is changing the behaviour of the default case, and throwing an error for the now-unsupported behaviour (in any places it's used that we haven't found) is better than silently returning fewer results than we had before. We'd have to document it clearly, of course -- probably in the exception.

(If `start` were before `rows` in the parameter list, I'd suggest removing the default value and making it mandatory.)